news/976.feature

```diff
@@ -0,0 +1 @@
+Override grammar now supports nested interpolations (ex: `python my_app.py 'key=${foo.${bar}}'`)
```

Comment:
Bump the version of Hydra to 1.1.0-dev1.
This is the first major change.

Reply:
Done in 416abb1b1